packages/components/alert/src/alert-icons.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  CloseIcon,
+  DangerIcon,
+  InfoCircleIcon,
+  SuccessIcon,
+  WarningIcon,
+} from "@nextui-org/shared-icons";
+
+export const AlertCloseIcon = (props: {color: string}) => {
+  switch (props.color) {
+    case "primary":
+      return <CloseIcon className="fill-current text-primary-200" height={20} width={20} />;
+
+    case "secondary":
+      return <CloseIcon className="fill-current text-secondary-200" height={20} width={20} />;
+
+    case "success":
+      return <CloseIcon className="fill-current text-success-200" height={20} width={20} />;
+
+    case "warning":
+      return <CloseIcon className="fill-current text-warning-200" height={20} width={20} />;
+
+    case "danger":
+      return <CloseIcon className="fill-current text-danger-200" height={20} width={20} />;
+
+    default:
+      return <CloseIcon className="fill-current text-default-400" height={20} width={20} />;
+  }
```

Comment:
I believe the style for `AlertIcon` should also be defined in `alert.ts`.

Reply:
Just to be sure, You only mean the close icon right?

Reply:
I think the same approach should be applied to AlertIcon as well.

Reply:
okay

Reply:
I have implemented these changes.

Reply:
Sorry if I wasn't clear. I think `className` should not be defined here.
Instead, the intent is to call it like the following examples.

https://github.com/nextui-org/nextui/blob/765dac53cfe6b04d1b1afda2410afb2ce3844f0f/packages/components/avatar/src/avatar.tsx#L45-L47

https://github.com/nextui-org/nextui/blob/765dac53cfe6b04d1b1afda2410afb2ce3844f0f/packages/components/modal/src/modal-content.tsx#L57-L59

Reply:
Just to be sure,
Instead of defining the `className` seperately for `closeIcon` slot, I should define them in `closeButton` itself and avoid the use of `getCloseIconProps`, right?

![image](https://github.com/user-attachments/assets/25cc7c90-d39a-4e41-9478-33e81e43951a)

Reply:
I have implemented the changes in my latest commit.
I rmeoved getCloseIconProps. I am calling it in a way similar to avatar component